ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -25,25 +72,17 @@ public final class Atomic<Value> {
 	}
 	
 	/// Initializes the variable with the given initial value.
-	public init(_ value: Value) {
+	public init(_ value: Value, mutex: MutexType = PosixThreadMutex()) {
```

Comment:
I'll repeat myself: I think this is a great refactor to take this out of `MutableProperty`, but I don't think this constructor should be `public` .